site/src/docs/components/highlight/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+slug: '/components/highlight'
+title: 'Highlight'
+navTitle: 'Highlight'
+---
+
+import { Highlight } from 'hds-react';
+import ExternalLink from '../../../components/ExternalLink';
+import PlaygroundPreview from '../../../components/Playground';
+import TabsLayout from './tabs.mdx';
+import InternalLink from '../../../components/InternalLink';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Usage
+
+### Example
+
+<PlaygroundPreview style={{ backgroundColor: 'var(--color-black-5)' }}>
+  <Highlight text="You may select an highlight from the article to be displayed here. Select an excerpt that you want the user to pay attention to." />
+</PlaygroundPreview>
+
+<PlaygroundPreview style={{ backgroundColor: 'var(--color-black-5)' }}>
+  <Highlight type="quote" text="You may select an highlight from the article to be displayed here. Select an excerpt that you want the user to pay attention to." />
+</PlaygroundPreview>
+
+### Principles
+
+- **Select an excerpt that you want the user to pay attention to.** 
+- Quote type wraps text inside quote marks
```

Comment:
I think this principles part needs some "food" as to when and why to use these, and when to use which one. I think there's something usable in the Jira ticket's description, at least for when to use specific types.

I think we can copy at least from Notification doc here:
`Highlights are designed to capture users' attention. ** Therefore, they should be used sparingly. **`

Reply:
Bump

Reply:
And maybe this would make it clearer about how to use quotes within the Highlight component:
- Use the prop `type="quote"` to transform the highlight text into a quote with an optional reference

If you put something like that, maybe remove the `- Quote type wraps text inside quote marks`?

Reply:
When using quotes, do we strongly recommend adding the reference prop?